### PR TITLE
opencompetition in md

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: docs/opencompetition.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Open Optimization Competition 2020
```
Comment (jrapin) [inline docs/opencompetition.md]: I would tend to call this file `opencompetition2020.md` instead, to be forward compatible. Then again, in 2021 we would probably delete it so it does not change much

File: docs/opencompetition.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+https://github.com/IOHprofiler/ for IOHprofiler [for IOHprofiler, please use the competition branch for your submissions]
+All supporting material should be uploaded together with the pull request. Links to arXiv papers etc are possible and welcome, but by no means mandatory. Keep in mind that a good description of your contribution increases the chance that jury members will understand and value your contribution.  
+All pull requests not yet merged on December 1 2019 or opened before June 1, 2020 are eligible for  the competition
+
+## Key principles:
+- Open source: no matter how good are the results, if they can not be reproduced or the code can not be checked this is not in the scope.
+- Reproducibility: if the code can not be run easily, it is also not in the scope.
+
+## Dates: 
+		Submission deadline is June 1, 2020, AoE
```
Comment (jrapin) [inline docs/opencompetition.md]: This does not render very well, maybe use bold instead? (using 2 stars "**" at each end of the text)

File: docs/opencompetition.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+Submissions made by members of the organizing committee or by employees of Facebook can not be awarded. Their coworkers can be rewarded (but only for work that does not involve organizers nor Facebook employees).
+
+## Details about the Submissions 
+All submissions are based on pull request, which are directly made to either one of the tools, via
+https://github.com/facebookresearch/nevergrad for Nevergrad 
+https://github.com/IOHprofiler/ for IOHprofiler. For submissions to IOHprofiler, please use the competition branch 
+
+## Recommended topics (non-exhaustive list)
+We identify the following list of topics for which we feel that great contributions are possible.
+##Improvements in any of the following optimization categories:
```
Comment (jrapin) [inline docs/opencompetition.md]: skip a line before the title, and a space after the `#` , otherwise it does not render well, also, a `#` is missing

**Comment by CarolaDoerr** [comment]
Jeremy, is it possible that I directly edit this file? I have now created a separate pull request, I believe -- sorry, I am somewhat lost (new to GitHub...)

**Comment by jrapin** [comment]
The simplest way for now is indeed to create a separate pull request on this branch, I just merged it now, your edits should be included

File: docs/opencompetition.md (hunk 1 above)
Comment (teytaud) [reply docs/opencompetition.md]: Done.

File: docs/opencompetition.md (hunk 2 above)
Comment (teytaud) [reply docs/opencompetition.md]: Done.

File: docs/opencompetition.md (hunk 3 above)
Comment (teytaud) [reply docs/opencompetition.md]: Done.

**Pull request merged**